react/scss/composers-app.scss

```diff
@@ -0,0 +1,31 @@
+@import '~bootstrap/dist/css/bootstrap.css';
+
+.pagination__header, .pagination__search, .pagination__suggestions, .pagination__content, .pagination__footer {
+  margin: 20px 0 20px 0;
+}
+
+.pagination-footer__separator {
```

Comment:
I'm still quite new to the BEM style for CSS. Could you elaborate what is the difference / advantage of `pagination-footer__separator` vs `pagination__footer__separator` as a class name? Thanks.